### DISCUSSION TITLE
漫展抓拍到两只虎皮与一个Coke

### DISCUSSION THREAD

**Discussion by biliqide** [share121/inter-knot#1505] [其他]
顺便一提漫展的帅哥美女很多欧😍![IMG20240811163422](https://github.com/user-attachments/assets/45ec02f4-3dae-4bbb-ad0e-e26fe56934eb)

**Comment by ZhuYuan-148** [comment]
比利？让狡兔屋负责人妮可来趟治安局，需要调查一下六分街售货机的事。
